packages/styles/scss/components/number-input/_number-input.scss

```diff
@@ -0,0 +1,479 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use '../../colors' as *;
+@use '../../config' as *;
+@use '../../motion' as *;
+@use '../../spacing' as *;
+@use '../../theme' as *;
+@use '../../type' as *;
+@use '../../utilities/skeleton' as *;
+@use '../../utilities/button-reset';
+@use '../../utilities/high-contrast-mode' as *;
+@use '../../utilities/focus-outline' as *;
+
+/// Number input styles
+/// @access public
+/// @group number-input
+@mixin number-input {
+  .#{$prefix}--number {
+    @include reset;
+
+    position: relative;
+    display: flex;
+    width: 100%;
+    flex-direction: column;
+  }
+
+  .#{$prefix}--number input[type='number'] {
+    @include type-style('body-short-01');
+    @include focus-outline('reset');
+
+    display: inline-flex;
+    width: 100%;
+    min-width: 9.375rem;
+    height: rem(40px);
```

Comment:
I think you'll need to bring in the `convert` utilities package for this 👀

Reply:
oh yes I'll update!